server/db/migrate/20150216190947_add_rhev_params.rb

```diff
@@ -0,0 +1,16 @@
+class AddRhevParams < ActiveRecord::Migration
```

Comment:
Since we haven't really started using the deployments migration yet, did you want to update the original migration (20150123165155_create_deployments.rb) versus adding a new one.  I know that is not ideal, but since only us devs are impacted, I do not mind rolling back and reapplying the migration.